versioned_docs/version-2.0/how_to_guides/monitoring/rules.mdx

```diff
@@ -43,11 +43,22 @@ You can create a filter as you normally would to filter traces in the project. F
 
 ![](../static/rules_filter.png)
 
-### Step 3: Define the sampling rate
+## (Optional) Step 3: Apply Rule to Past Runs
+
+When creating a new rule, you can apply the rule to past runs as well. To do this, select **Apply to Past Runs** checkbox and enter **Backfill From** date as the
+start date to apply the rule.
+
+This will start from the **Backfill From** date and apply the run rules until it is caught up with the latest runs.
+
+![](../static/rules_past_runs.png)
+
+Note that you will have to expand the date range for logs if you wanted to look at the progress of the backfill, see [View logs for your automations](./how_to_guides/monitoring/rules#view-logs-for-your-automations) for details.
```

Comment:
This is broken - it should be something like ./#view-logs-for-your-automations but forget exact syntax

Reply:
Ah ok good catch, ./rules/#... seems to work